back-office/multistoregeneralspecs.md

```diff
@@ -14,6 +14,12 @@ From 1.7.8, while creating a shop or a group, the user can select a color as des
 This color will be displayed in the multistore header for the selected shop or shop group.<br/>
 It will be also displayed before the name of each shop / group in the context drop-down.
 
+**User acceptance test**
```

Comment:
Nothing to do with UAT, sorry: you wrote in the 'What' section above that specific behaviors could be found in `back-office/multistorespecialsspecs.md` but they actually are located in `broader-topics/multistorespecialsspecs.md`. :-)

Reply:
Nice catch, I just changed it !